composeApp/src/androidMain/kotlin/io/github/couchtracker/db/tmdb/TmdbCacheDb.kt

```diff
@@ -0,0 +1,29 @@
+package io.github.couchtracker.db.tmdb
+
+import android.content.Context
+import app.cash.sqldelight.adapter.primitive.IntColumnAdapter
+import app.cash.sqldelight.async.coroutines.synchronous
+import app.cash.sqldelight.driver.android.AndroidSqliteDriver
+
+object TmdbCacheDb {
+    fun get(context: Context): TmdbCache {
+        return TmdbCache(
+            driver = AndroidSqliteDriver(
+                // See https://github.com/cashapp/sqldelight/issues/5058
+                TmdbCache.Schema.synchronous(),
+                context,
+                "tmdb-cache.db",
+            ),
+            MovieDetailsCacheAdapter = MovieDetailsCache.Adapter(
+                tmdbIdAdapter = IntColumnAdapter,
```

Comment:
Wow! Does SQLDelight require explicitly specifying the adapter even for simple Ints? That seems unecessary.